crosshair/core_test.py

```diff
@@ -24,49 +24,45 @@
 from crosshair.util import set_debug
 
 
-
-
-
-
-
-
 #
 # Begin fixed line number area.
```

Comment:
Right, so this "fixed line number area" is an odd/hacky case - I have these spare newlines here so that I can just cut out some when import lines are added without changing the tests.

I imagine we could figure out a different way to handle this though. Maybe base the line numbers in the tests as offsets from the line numbers derivable from the function objects themselves.

Reply:
I think there were so few breaking tests that I would suggest just to update them manually at the moment?